### PR TITLE
version 2

### PR DESCRIPTION
**Pull request by richardbedwell**

### CODE REVIEW

**Comment by richardbedwell** [comment]
i made a few changes that i suggest making

firstly i messed up the csv file a bit, throwing in some bad data rows so we can see how they deal with that. 

also reworded the requirements a bit, removed allowing javascript and allowed the candidate to explain how it can scale to large volumes of data rather than necessarily having the local script able to deal with it. that should allow more of the time limit to deal with testing and the like which will give us a better idea of how the candidate works.

File: README.md [hunk 1]
```diff
@@ -37,12 +37,14 @@ another microservice), the following requirements need to be met.
...
 
-- Automated processes to ingest and store the data on an on-demand basis
+- Automated or on-demand processes to ingest and store the data
+- Test the quality of the data
 - A service that is able to provide the following functionality:
   - Return the total amount processed by acquirer and by currency for a given time range
   - Inform about the status of the data ingestion
-- The solution should be scalable to 100 million entries. It is encouraged to simplify the data by a data model. Please add proof that the solution is scalable.
-- The solution should be written in Python or JavaScript using an SQL database
+- The solution should be scalable to 100 million entries. Please explain how your process can deal with this volume and/or changes that you would make to overcome any limiations.
```
Comment (vfaramond) [inline README.md:45]: limitations *

File: README.md [hunk 2]
```diff
@@ -37,12 +37,14 @@ another microservice), the following requirements need to be met.
...
+- The solution should be written in Python using a sutiable database.
```
Comment (vfaramond) [inline README.md:46]: suitable *

**Review by vfaramond — CHANGES_REQUESTED** [review]

File: data/sample_acquirer_a.csv [hunk 3]
```diff
@@ -184,7 +184,7 @@ Fintech Limited,Fintech Limited,Acquirer 1,2020-08-19T13:08:23.685Z,Declined,Aut
...
 Fintech Limited,Fintech Limited,Acquirer 1,2020-08-19T02:28:24.030Z,Declined,Void,MasterCard,Yes,Debit,Georgia,Declined,bb9ac05e-96df-4d35-a95b-de4cb81495a0,No,7778,6494714d-9e8f-46e7-ad4d-056714684f29,RON,18692,-467.3,0.00,0.00,0.00,0.00,18224.7,EUR,3861.74,0.00,
 Fintech Limited,Fintech Limited,Acquirer 1,2020-08-19T04:41:51.571Z,Declined,Sale,MasterCard,Yes,Credit,Japan,Declined,913874a1-f5a6-4f77-92a9-116447cd6405,No,2611,3c2db77d-4621-4221-9193-7189ced45cbc,HUF,14224,-355.6,0.00,0.00,0.00,0.00,13868.4,EUR,40.66,0.00,
-Fintech Limited,Fintech Limited,Acquirer 1,2020-08-19T15:23:39.068Z,Declined,Sale,MasterCard,No,Debit,Pakistan,Declined,2c37925f-2d82-4ca6-8673-8854e68c2a4b,No,6733,a50253a3-287c-464c-8722-faace5584edf,SEK,4492,-112.3,0.00,0.00,0.00,0.00,4379.7,EUR,435.52,0.00,
+Fintech Limited,2020-08-19T15:23:39.068Z,Declined,Sale,MasterCard,No,Debit,Pakistan,Declined,2c37925f-2d82-4ca6-8673-8854e68c2a4b,No,6733,a50253a3-287c-464c-8722-faace5584edf,SEK,4492,-112.3,0.00,0.00,0.00,0.00,4379.7,EUR,435.52,0.00,
```
Comment (nazwa) [inline data/sample_acquirer_a.csv:187]: You're so mean! 😡 🤣

**Pull request merged**